roles/nodepool/tasks/main.yml

```diff
@@ -93,8 +106,22 @@
   synchronize:
     src: files/etc/nodepool/
     dest: /etc/nodepool/
-    rsync_opts:
-      - "--chown=nodepool:nodepool"
+# The option below is amazing, but requires rsync 3.1+
+#    rsync_opts:
+#      - "--chown=nodepool:nodepool"
```

Comment:
we should look at pulling these out into their own git repo

Reply:
Agreed. This solution, btw, SUCKS. It changes things every time. I'd much rather use --chown. I might even be willing to chase down upgrade rsync for centos 7. But.. this was the quickest way I got to a working nodepool on a short schedule. Open to other suggestions too.